plugins/services/src/js/utils/ServiceUtil.js

```diff
@@ -114,6 +114,45 @@ const ServiceUtil = {
       service.getLabels &&
       service.getLabels().DCOS_COMMONS_API_VERSION != null
     );
+  },
+
+  /**
+   * this function extracts webURL logic from all structs to keep it in one place
```

Comment:
This comment refers to the refactoring (where the code was before), it would be cooler if you could describe what it does and why

Reply:
fair point 👍